### ISSUE TITLE
b018 time is off

### ISSUE THREAD

**Issue by PeteyHall** [dpitts/f16v3_issues#48]
On the main page, the Controller Time is wrong - it appears to reset to 1-1-01 00:00:00 on each reboot.

If I click on Misc settings, the time is correct under Set Time but if I click Set Time, nothing happens and the time on the controller is still incorrect.

**Comment by dpitts** [comment]
You need rtc battery.

On Oct 21, 2018 11:27 AM, PeteyHall <notifications@github.com> wrote:

On the main page, the Controller Time is wrong - it appears to reset to 1-1-01 00:00:00 on each reboot.

If I click on Misc settings, the time is correct under Set Time but if I click Set Time, nothing happens and the time on the controller is still incorrect.

**Comment by dpitts** [comment]
PeteyHall was the controller time counting on Misc Setting Page even though it was wrong?

**Comment by PeteyHall** [comment]
@dpitts Yes the Current Time counts up from the last time power was applied to the controller.  Even if I just reboot the controller, it'll continue to count from where it was before the reboot.

If I remove power and reboot then the clock goes all the way back.

**Comment by PeteyHall** [comment]
FYI, after the factory reset, I'm able to set the time but there's something wrong.  Today is 10-21-18 but when I click Set Time (it has the right date/time in there), it sets the right time now but the date is set to 00-21-18, so it seems to be having an issue with a 2-month date.

**Comment by PeteyHall** [comment]
![image](https://user-images.githubusercontent.com/44344796/47272764-394d3680-d558-11e8-803c-281ca96639d5.png)

**Comment by PeteyHall** [comment]
![image](https://user-images.githubusercontent.com/44344796/47272781-6bf72f00-d558-11e8-9386-b48a702a11c6.png)

**Comment by dpitts** [comment]
Crash when setting time when not in player mode (E131/Artnet)

**Comment by dpitts** [comment]
dsdsd

**Comment by JonB256** [comment]
I see - will wait for b019 to be posted.
it was odd - 10 (October) displayed as 00, 11 (November) as 01 and 12 (December) as 02.
Also need to replace my battery. Time is not surviving a reboot.

**Comment by PeteyHall** [comment]
If you want me to test 19, send it over and i'll try it tonight.
